speechbrain/nnet/ligru/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Package containing LiGRU Cell with CuPy.
```

Comment:
Add a bit more of context. Why do we need this package, what is he bringing and under what conditions? Do we need to install something first?